utils.py

```diff
@@ -0,0 +1,10 @@
+def preprocess_df(df, country):
```

Comment:
Consider renaming this to `filter_country_sort_year` as `preprocess_df` is a rather generic name